R/execution.R

```diff
@@ -553,7 +553,8 @@ if (conceptCheckThresholdLoc == "default"){
     
     if (!is.na(checkResults[i,]$ERROR)) {
       checkResults[i,]$FAILED <- 1
-    } else if (is.na(thresholdValue)) {
+    } else if (is.na(thresholdValue) | thresholdValue == 0) {
```

Comment:
i think this got added in when i switched to target develop instead of main - please let me know if i should revert this change

Reply:
I think that is fine.